### PR TITLE
Add parameters `$useBootstrap`, `$checkEvents` and `$configGroupPostfix` to the `HttpApplicationRunner`

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ✔️
| Fixed issues  | https://github.com/yiisoft/app/issues/271

### CODE REVIEW

**Review by samdark — CHANGES_REQUESTED** [review]
README should be updated.

File: README.md [hunk 1]
```diff
@@ -43,44 +43,55 @@ use Yiisoft\Yii\Runner\Http\HttpApplicationRunner;
...
 
-(new HttpApplicationRunner(__DIR__, $_ENV['YII_DEBUG'], $_ENV['YII_ENV']))->run();
+(new HttpApplicationRunner(
+    rootPath: __DIR__, 
+    debug: $_ENV['YII_DEBUG'],
+    checkEvents: $_ENV['YII_DEBUG'],
+    environment: $_ENV['YII_ENV']
+))->run();
 ```
+By default, HTTP runner .
```
Comment (samdark) [inline README.md:53]: ?

File: src/HttpApplicationRunner.php [hunk 2]
```diff
@@ -37,25 +37,53 @@ final class HttpApplicationRunner extends ApplicationRunner
...
+     * @param string $bootstrapGroup The bootstrap configuration group name.
+     * @param string $eventsGroup The events' configuration group name.
+     * @param string $diGroup The container definitions' configuration group name.
+     * @param string $diProvidersGroup The container providers' configuration group name.
+     * @param string $diDelegatesGroup The container delegates' configuration group name.
+     * @param string $diTagsGroup The container tags' configuration group name.
+     * @param string $paramsGroup The config parameters group name.
+     * @param array $nestedParamsGroups Configuration group names that included to config parameters group. This is
+     * needed for recursive merge parameters.
+     * @param array $nestedEventsGroups Configuration group names that included to events' configuration group. This is
```
Comment (samdark) [inline src/HttpApplicationRunner.php:51]:
```suggestion
     * @param array $nestedEventsGroups Configuration group names that included into events' configuration group. This is
```

File: src/HttpApplicationRunner.php [hunk 3]
```diff
@@ -37,25 +37,53 @@ final class HttpApplicationRunner extends ApplicationRunner
...
+     * needed for reverse and recursive merge events' configurations.
```
Comment (samdark) [inline src/HttpApplicationRunner.php:52]:
```suggestion
     * needed for reverse and recursive merge of events' configurations.
```

**Review by samdark — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vjik) [reply README.md]: fixed

File: src/HttpApplicationRunner.php [hunk 4]
```diff
@@ -37,25 +37,53 @@ final class HttpApplicationRunner extends ApplicationRunner
...
-     * checked in debug mode only.
+     * @param bool $checkEvents Whether check events' configuration.
      * @param string|null $environment The environment name.
+     * @param string $bootstrapGroup The bootstrap configuration group name.
+     * @param string $eventsGroup The events' configuration group name.
+     * @param string $diGroup The container definitions' configuration group name.
+     * @param string $diProvidersGroup The container providers' configuration group name.
+     * @param string $diDelegatesGroup The container delegates' configuration group name.
+     * @param string $diTagsGroup The container tags' configuration group name.
+     * @param string $paramsGroup The config parameters group name.
```
[the remark this answers is not in the document]
Comment (vjik) [reply src/HttpApplicationRunner.php]: done

File: src/HttpApplicationRunner.php [hunk 5]
```diff
@@ -37,25 +37,53 @@ final class HttpApplicationRunner extends ApplicationRunner
     /**
      * @param string $rootPath The absolute path to the project root.
      * @param bool $debug Whether the debug mode is enabled.
-     * @param string $paramsConfigGroup The config parameters group name.
-     * @param string $containerConfigGroup The container configuration group name.
-     * @param string|null $bootstrapGroup The bootstrap configuration group name.
-     * @param string|null $eventsGroup The event configuration group name to check. The configuration of events is
-     * checked in debug mode only.
+     * @param bool $checkEvents Whether check events' configuration.
```
[the remark this answers is not in the document]
Comment (vjik) [reply src/HttpApplicationRunner.php]: done

File: src/HttpApplicationRunner.php [hunk 6]
```diff
@@ -37,25 +37,53 @@ final class HttpApplicationRunner extends ApplicationRunner
...
      * @param string|null $environment The environment name.
+     * @param string $bootstrapGroup The bootstrap configuration group name.
+     * @param string $eventsGroup The events' configuration group name.
+     * @param string $diGroup The container definitions' configuration group name.
+     * @param string $diProvidersGroup The container providers' configuration group name.
+     * @param string $diDelegatesGroup The container delegates' configuration group name.
+     * @param string $diTagsGroup The container tags' configuration group name.
+     * @param string $paramsGroup The config parameters group name.
+     * @param array $nestedParamsGroups Configuration group names that included to config parameters group. This is
```
[the remark this answers is not in the document]
Comment (vjik) [reply src/HttpApplicationRunner.php]: done

File: src/HttpApplicationRunner.php (hunk 2 above)
Comment (vjik) [reply src/HttpApplicationRunner.php]: done

File: src/HttpApplicationRunner.php (hunk 3 above)
Comment (vjik) [reply src/HttpApplicationRunner.php]: done

File: src/HttpApplicationRunner.php [hunk 7]
```diff
@@ -37,25 +37,53 @@ final class HttpApplicationRunner extends ApplicationRunner
...
+     * needed for recursive merge parameters.
```
[the remark this answers is not in the document]
Comment (vjik) [reply src/HttpApplicationRunner.php]: done

File: README.md [hunk 8]
```diff
@@ -43,44 +43,54 @@ use Yiisoft\Yii\Runner\Http\HttpApplicationRunner;
...
+
+`$diProvidersGroup` — the container providers' configuration group name.
+
+`$diDelegatesGroup` — the container delegates' configuration group name.
+
+`$diTagsGroup` — the container tags' configuration group name.
+
+`$paramsGroup` — the config parameters group name.
+
+`$nestedParamsGroups` — configuration group names that included to config parameters group. This is needed for recursive
```
Comment (samdark) [inline README.md:85]:
```suggestion
`$nestedParamsGroups` — configuration group names that are included into config parameters group. This is needed for recursive
```

File: README.md [hunk 9]
```diff
@@ -43,44 +43,54 @@ use Yiisoft\Yii\Runner\Http\HttpApplicationRunner;
...
+merge parameters.
+ 
+`$nestedEventsGroups` — configuration group names that included to events' configuration group. This is needed for
```
Comment (samdark) [inline README.md:88]:
```suggestion
`$nestedEventsGroups` — configuration group names that are included into events' configuration group. This is needed for 
```

File: src/HttpApplicationRunner.php [hunk 10]
```diff
@@ -37,25 +37,53 @@ final class HttpApplicationRunner extends ApplicationRunner
...
+     * @param bool $checkEvents Whether to check events' configuration.
      * @param string|null $environment The environment name.
+     * @param string $bootstrapGroup The bootstrap configuration group name.
+     * @param string $eventsGroup The events' configuration group name.
+     * @param string $diGroup The container definitions' configuration group name.
+     * @param string $diProvidersGroup The container providers' configuration group name.
+     * @param string $diDelegatesGroup The container delegates' configuration group name.
+     * @param string $diTagsGroup The container tags' configuration group name.
+     * @param string $paramsGroup The configuration parameters group name.
+     * @param array $nestedParamsGroups Configuration group names that included into configuration parameters group.
```
Comment (samdark) [inline src/HttpApplicationRunner.php:49]:
```suggestion
     * @param array $nestedParamsGroups Configuration group names that are included into configuration parameters group.
```

File: src/HttpApplicationRunner.php [hunk 11]
```diff
@@ -37,25 +37,53 @@ final class HttpApplicationRunner extends ApplicationRunner
...
+     * This is needed for recursive merging of parameters.
+     * @param array $nestedEventsGroups Configuration group names that included into events' configuration group. This
```
Comment (samdark) [inline src/HttpApplicationRunner.php:51]:
```suggestion
     * @param array $nestedEventsGroups Configuration group names that are included into events' configuration group. This
```

**Review by samdark — APPROVED** [review]

**Pull request merged**